support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.jsx

```diff
@@ -284,23 +292,14 @@ function WeeklyCheckoutForm(props: PropTypes) {
             productPrices={props.productPrices}
             selected={props.billingPeriod}
           />
-          <PaymentMethodSelector
-            country={props.billingCountry}
-            paymentMethod={props.paymentMethod}
-            setPaymentMethod={props.setPaymentMethod}
-            onPaymentAuthorised={props.onPaymentAuthorised}
-            validationError={firstError('paymentMethod', props.formErrors)}
-            csrf={props.csrf}
-            currencyId={props.currencyId}
-            payPalHasLoaded={props.payPalHasLoaded}
-            formIsValid={props.formIsValid}
-            validateForm={props.validateForm}
-            isTestUser={props.isTestUser}
-            setupRecurringPayPalPayment={props.setupRecurringPayPalPayment}
-            amount={price.price}
-            billingPeriod={props.billingPeriod}
-            allErrors={[...props.billingAddressErrors, ...props.deliveryAddressErrors, ...props.formErrors]}
-          />
+          <FormSection title="How would you like to pay?">
```

Comment:
Don't we need the `paymentMethods.length > 1 ?` check here?

Reply:
Yes, thanks Rupert!